nengo_loihi/tests/test_interneurons.py

```diff
@@ -0,0 +1,72 @@
+import nengo
+import numpy as np
+import pytest
+
+from nengo_loihi.builder import (
+    Model, NoisyInterneurons, OnOffInterneurons,
+    Preset5Interneurons, Preset10Interneurons,
+)
+
+
+@pytest.mark.parametrize('interneurons, tolerance', [
+    (OnOffInterneurons(), 0.32),
+    (NoisyInterneurons(5), 0.11),
+    (NoisyInterneurons(10), 0.10),
+    (Preset5Interneurons(), 0.03),
+    (Preset10Interneurons(), 0.03),
+])
+def test_add_inputs(interneurons, tolerance, Simulator, seed, plt):
+    sim_time = 2.
+    pres_time = 0.5
+    eval_time = 0.25
+
+    stim_fn_a = nengo.processes.Piecewise({
+        t: (2*(i % 2) - 1) * 0.4
+        for i, t in enumerate(np.arange(0, sim_time, pres_time))})
+    stim_fn_b = stim_fn_a
+
+    with nengo.Network(seed=seed) as model:
+        stim_a = nengo.Node(stim_fn_a)
+        stim_b = nengo.Node(stim_fn_b)
+
+        a = nengo.Ensemble(n_neurons=100, dimensions=1)
+        b = nengo.Ensemble(n_neurons=100, dimensions=1)
+
+        nengo.Connection(stim_a, a)
+        nengo.Connection(stim_b, b)
+
+        c = nengo.Ensemble(n_neurons=100, dimensions=1)
+        nengo.Connection(a, c)
+        nengo.Connection(b, c)
+
+        out_synapse = nengo.Alpha(0.03)
+        stim_synapse = out_synapse.combine(
+            nengo.Alpha(0.005)).combine(
+                nengo.Alpha(0.005))
+        p_stim_a = nengo.Probe(stim_a, synapse=stim_synapse)
+        p_stim_b = nengo.Probe(stim_b, synapse=stim_synapse)
+        p_c = nengo.Probe(c, synapse=out_synapse)
+
+    build_model = Model()
+    build_model.inter_neurons = interneurons
+
+    with Simulator(model, model=build_model) as sim:
+        sim.run(sim_time)
+
+    t = sim.trange()
+    tmask = np.zeros(t.shape, dtype=bool)
+    for pres_t in np.arange(0, sim_time, pres_time):
+        t0 = pres_t + pres_time - eval_time
+        t1 = pres_t + pres_time
+        tmask |= (t >= t0) & (t <= t1)
+
+    target = sim.data[p_stim_a] + sim.data[p_stim_b]
+
+    error = np.abs(sim.data[p_c][tmask] - target[tmask]).mean()
+    print(error)
```

Comment:
I'm guessing this was here to get good tolerance values initially? I would probably remove it in lieu of maybe something like `plt.title("error=%g" % error)`

Reply:
Partly to get good initial values, but also because this is somewhat a benchmark in addition to a test. I use this test to make changes to different interneuron classes and see how it affects the performance.

My plan was to use the logger fixture once the PR where I add that gets merged, but I still have some problems with the logger (i.e. it would be nice to be able to just print everything that it logs to the screen, with a command line option like `--log-to-screen`).